src/nvme/fabrics.c

```diff
@@ -47,19 +47,16 @@
 const char *nvmf_dev = "/dev/nvme-fabrics";
 
 /**
- * strchomp() - Strip trailing white space
+ * strchomp() - Strip trailing spaces
  * @str: String to strip
  * @max: Maximum length of string
  */
 static void strchomp(char *str, int max)
 {
 	int i;
 
-	for (i = max - 1; i >= 0; i--) {
-		if (str[i] != '\0' && str[i] != ' ')
```

Comment:
I digged into the history and found the source of the NULL check:

e48ba33d19ae ("fabrics: Handle space-padded TRSVCID and TRADDR fields")
https://github.com/linux-nvme/nvme-cli/commit/e48ba33d19ae

I suppose  @rolandd is fine with this change ;)